src/styles/blocks/watch.scss

```
@@ -0,0 +1,59 @@
+.watch {
+  position: relative;
+
+  width: $watch-height;
+  height: $watch-height;
+
+  &__center {
+    position: absolute;
+    top: 50%;
+    left: 50%;
+    z-index: 1;
+  
+    width: 6%;
+    height: 6%;
```

```suggestion
    width: 6%;
    height: 6%;
```
Instead of using these 'magic numbers' it would be better to use pixels in such cases